bundlesize.config.json

```
@@ -0,0 +1,56 @@
+{
+  "files": [
+    {
+      "path": "static/build/editions-table.*.js",
+      "maxSize": "16.8KB"
+    },
+    {
+      "path": "static/build/graphs.*.js",
+      "maxSize": "17.2KB"
```

@cdrini there is a conflict within package.json otherwise LGTM
```suggestion
      "maxSize": "17.3KB"
```

---

Fixed :+1: